src/Button/Button.story.tsx

```diff
@@ -0,0 +1,61 @@
+import * as React from 'react';
+import { DefaultButton } from '.';
+import { storiesOf } from '@storybook/react';
+
+const stories = storiesOf('Button', module);
+
+stories.add('Default', () => {
+  return (
+    <DefaultButton className={'usa-button'}>
+      {'Click Me'}
+    </DefaultButton >
+  )
+});
+
+stories.add('Secondary Color', () => {
+  return (
+    <DefaultButton className={'usa-button usa-button--secondary'}>
```

Comment:
Since 'usa-button' is necessary for every button, you can bake this classname into the `<DefaultButton>` component and expand the classes for the other buttons.  In fact, we could define other button components with the appropriate classnames baked in so that the implementer does not need to worry about which class name is necessary to theme the buttons, they just have to focus on which button to import.

Reply:
@xylocopaCaerulea so you're saying I should make a separate file for every class of button? I can do that, it seems cleaner as well.